### PR TITLE
Add watchos support to podspec

### PR DESCRIPTION
**Pull request by rdgborges**

### CODE REVIEW

File: KZPropertyMapper.podspec [hunk 1]
```diff
@@ -1,14 +1,15 @@
 Pod::Spec.new do |s|
   s.name         = "KZPropertyMapper"
-  s.version      = "2.8"
```
Comment (adelinofaria) [inline KZPropertyMapper.podspec]: wait wat? Shouldn't be a branch from 2.6 tag?

Comment (crsantos) [reply KZPropertyMapper.podspec]: Our developments weren't merged into master, but I advise you to work on top of `2.6.1` - that's the latest tag we support. We don't have the time right now to bump to 2.9 without proper tests. So I advise you to create `2.6.2`.

**Comment by adelinofaria** [comment]
Closing this PR, I'm opening one with same changes but on top of the correct base commit.

File: KZPropertyMapper.podspec (hunk 1 above)
Comment (rdgborges) [reply KZPropertyMapper.podspec]: Sorry for the mistake, guys. My intent was to open the PR to 2.6.1. Thank you so much for your help with that! 👍

**Pull request closed without merging**